test/functional/p2p_filter.py

```diff
@@ -0,0 +1,107 @@
+#!/usr/bin/env python3
+# Copyright (c) 2020 The Bitcoin Core developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+"""
+Test BIP 37
+"""
+
+from test_framework.messages import (
+    MSG_BLOCK,
+    MSG_FILTERED_BLOCK,
+    msg_getdata,
+    msg_filterload,
+)
+from test_framework.mininode import (
+    P2PInterface,
+)
+from test_framework.test_framework import BitcoinTestFramework
+
+
+class FilterNode(P2PInterface):
+    # This is a P2SH watch-only wallet
+    watch_script_pubkey = 'a914ffffffffffffffffffffffffffffffffffffffff87'
+    # The initial filter with just the above scriptPubKey added
+    watch_filter_init = msg_filterload(
+        data=
+        b'@\x00\x08\x00\x80\x00\x00 \x00\xc0\x00 \x04\x00\x08$\x00\x04\x80\x00\x00 \x00\x00\x00\x00\x80\x00\x00@\x00\x02@ \x00',
+        nHashFuncs=19,
+        nTweak=0,
+        nFlags=1,
+    )
```

Comment:
Could add a short comment on what this filter's dimensions are based upon (w.r.t. target parameters number of elements N, false positive rate P)?

Reply:
Thanks, extended the comment to state the filter parameters.